Add physio and sensor session models and associations

DhruvK278 opened the pull request:
Introduces PhysioSession and SensorSessionLog Sequelize models with their respective associations to Patient and Doctor entities. Updates SQL schema for exercise_sessions with new fields and constraints. Also updates CORS origin in server configuration.

<img width="186" height="249" alt="image" src="https://github.com/user-attachments/assets/05be995f-8aa6-4ddf-9cff-9a057afa7812" />

DhruvK278 committed:
Add physio and sensor session models and associations

Introduces PhysioSession and SensorSessionLog Sequelize models with their respective associations to Patient and Doctor entities. Updates SQL schema for exercise_sessions with new fields and constraints. Also updates CORS origin in server configuration.

rohan-pandeyy commented on Backend/models/physio.model.js:
```diff
@@ -0,0 +1,43 @@
+      defaultValue: DataTypes.UUIDV4,
+      primaryKey: true,
+    },
+    patient_id: {
```
Please change patient_id to camelCase
ie `patientId`

rohan-pandeyy commented on Backend/models/physio.model.js:
```diff
@@ -0,0 +1,43 @@
+      type: DataTypes.UUID,
+      allowNull: false,
+    },
+    therapist_id: {
```
Kindly change to camelCase

rohan-pandeyy commented on Backend/models/sensor_sess.model.js:
```diff
@@ -0,0 +1,59 @@
+      defaultValue: DataTypes.UUIDV4,
+      primaryKey: true,
+    },
+    patient_id: {
```
Kindly change to camelCase

rohan-pandeyy commented on Backend/models/sensor_sess.model.js:
Kindly change all the snake_cases to camelCase

rohan-pandeyy commented on Backend/models/sql_schema.sql:
You may remove the `exercise_sessions` table completely from the file, it's not required

rohan-pandeyy commented on Backend/server.js:
```diff
@@ -22,7 +22,7 @@ app.use(
     origin: [
       'http://localhost:3000', // Allow requests from your frontend
-      'https://b71415fb3c10.ngrok-free.app/api/auth/register', //ngrok signup url here
+      'https://a8cf18283436.ngrok-free.app/api/auth/register', //ngrok signup url here
```
I'd suggest you replace this link with a placeholder of some sort.

rohan-pandeyy reviewed (changes requested):
Kindly go through these requested changes, once done, the PR can be merged to master

DhruvK278 committed:
Update model field names and remove exercise_sessions table

Renamed fields in physio.model.js and sensor_sess.model.js to use camelCase for consistency. Removed the unused exercise_sessions table from sql_schema.sql. Updated ngrok URL placeholder in server.js for improved configuration clarity.

DhruvK278 committed:
snake case  camelCase

rohan-pandeyy commented:
Closes #11
